pages/common/shc.md

```diff
@@ -0,0 +1,19 @@
+# shc
+
+> Generic shell script compiler.
+
+- Compile a shell script:
+
+`shc -f {{script}}`
+
+- Compile a shell script and specify an output:
+
+`shc -f {{script}} -o {{outputbinary}}`
+
+- Compile a shell script and set an expiration date for the executable:
```

Comment:
Just curious - what does it mean to have an expiration date for an executable ?

Reply:
You can set the binary to not be able to be executed after a set amount of time. `shc` is sometimes used for obfuscation to protect proprietary code. It allows you to prevent your software from being stolen in case the binary ends up in the wrong hands.

Reply:
got it. thanks.